### PR TITLE
Use int zero (0) for never in unlock_time setting for pam_faillock

### LINKED ISSUE #1606: super/sub compliance for accounts_passwords_pam_faillock_unlock_time.xml

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Move faillock variables to correct directory
- Use int zero (0) for lifetime lockout

#### Rationale:

- Originally, using int zero (0) for permanent lock out wasn't an option. Man pages now reference using int zero (0) for lifetime lockout, so let's use it.

- Fixes #1606

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/oval/shared.xml [hunk 1]
```diff
@@ -40,35 +40,35 @@
...
   <ind:textfilecontent54_object id="object_accounts_passwords_pam_faillock_preauth_unlock_time_password-auth" version="2">
     <ind:filepath>/etc/pam.d/password-auth</ind:filepath>
-    <ind:pattern operation="pattern match">^\s*auth\s+(?:(?:required))\s+pam_faillock\.so\s+preauth.*unlock_time=([a-z0-9]*).*$</ind:pattern>
+    <ind:pattern operation="pattern match">^\s*auth\s+(?:(?:required))\s+pam_faillock\.so\s+preauth.*unlock_time=([0-9]*).*$</ind:pattern>
     <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
   </ind:textfilecontent54_object>
 
   <ind:textfilecontent54_state id="state_accounts_passwords_pam_faillock_unlock_time_system-auth" version="2">
-    <ind:subexpression datatype="string" operation="equals" var_ref="var_accounts_passwords_pam_faillock_unlock_time" />
+    <ind:subexpression datatype="int" operation="less than or equal" var_ref="var_accounts_passwords_pam_faillock_unlock_time" />
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/oval/shared.xml]: Good catch!

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/rule.yml [hunk 2]
```diff
@@ -58,6 +59,6 @@ ocil_clause: 'unlock_time is less than the expected value'
 ocil: |-
     To ensure the failed password attempt policy is configured correctly, run the following command:
     <pre>$ grep pam_faillock /etc/pam.d/system-auth</pre>
-    The output should show <tt>unlock_time=&lt;some-large-number&gt;</tt> or <tt>never</tt>.
+    The output should show <tt>unlock_time=&lt;some-large-number&gt;</tt> or <tt>0</tt> for <tt>never</tt>.
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/rule.yml]: Can ``0`` be used across all RHEL versions, or should RHEL 6 still have ``never``?

Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/rule.yml]: @shawndwells looks like this was also finally fixed in the latest version of RHEL6

Comment (shawndwells) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/rule.yml]: Thanks for double checking.

**Pull request merged**